frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/CheckOutBranchAction.java

```diff
@@ -30,7 +31,7 @@
  *  <li>{@link DataKeys#KEY_SELECTED_VCS_REPOSITORY}</li>
  * </ul>
  */
-public class CheckOutBranchAction extends AnAction {
+public class CheckOutBranchAction extends AnAction implements DumbAware {
```

Comment:
This action wasn't `DumbAware` so far... was there any reason or it's just been overlooked?

Reply:
Nope, it's `dumbAware` :+1: